### ISSUE TITLE
Support for data.frames

### ISSUE THREAD

**Issue by gasparl** [WillemSleegers/tidystats#24]
How about adding support for any sort of `data.frame` (and perhaps `matrix`)? This is really simple and would cover any conceivable statistics that can be put in a table format. I already have a helper function for it ([df_to_group](https://github.com/gasparl/tidystats/blob/ce9d6423a0eeb45830a504aad70b26b3243e0725/R/helper_functions.r#L85); it was needed for metafor's `data.frame` outputs), so we would just need to wrap it under  a `tidy_stats` class.

**Comment by WillemSleegers** [comment]
That makes sense. Although it will be a bit tricky to figure out the symbols and potential subscripts. Although we could just omit those things.

**Comment by gasparl** [comment]
Yes, it could be either omitted or added as optional via a parameter. (I already have a short list of replacements for "frequently occurring" symbols, see [here](https://github.com/gasparl/tidystats/blob/ce9d6423a0eeb45830a504aad70b26b3243e0725/R/helper_functions.r#L117); this could be expanded and used for this purpose.)

**Comment by WillemSleegers** [comment]
Good idea. It could be an optional argument that takes either a vector or a list. A vector could be provided in case the user wants to provide a symbol/subscript for all stats, or a list in case only some of the statistics should have a symbol and/or subscript.

For example: `df_to_group(df, symbols = list("estimate" = "t"))`

**Comment by WillemSleegers** [comment]
We should call the function something else though. Perhaps `df_to_stats()` or `tidy_df()`.

**Comment by gasparl** [comment]
As discussed, this became part of the `add_stats` function (with additional optional parameters) rather than a function on its own; see  implemented [here](https://github.com/gasparl/tidystats/commit/f38faaca617b50d87aa822ddcfe85b750a644de0#diff-db51d06fd471546ced907fc2f5e90f469266eb7ffec1ec98002c4e56fbb65b7b). I guess a short description of the optional parameters should be given somewhere in the documentation. Let me know if I should add it (and if yes, where).
